lib/generate_localized.dart

```diff
@@ -72,6 +72,20 @@ class MessageGeneration {
   /// for the [translations] in [locale] and put it in [targetDir].
   void generateIndividualMessageFile(String basicLocale,
       Iterable<TranslatedMessage> translations, String targetDir) {
+    final content = generateIndividualMessageFileContent(
+        basicLocale, translations);
+
+    // To preserve compatibility, we don't use the canonical version of the
+    // locale in the file name.
+    var filename = path.join(
+        targetDir, "${generatedFilePrefix}messages_$basicLocale.dart");
+    new File(filename).writeAsStringSync(content);
```

Comment:
Current Dart style prefers to omit new. And if we're doing current Dart style probably change the " to '